### DISCUSSION TITLE
Does podman have cluster mode?

### DISCUSSION THREAD

**Discussion by ThorManager** [containers/podman#12886] [Q&A]
Is there a cluster mode? Because our environment is not suitable for k8s, are there other cluster solutions? Such as docker's swarm？

**Comment by afbjorklund** [comment]
There is https://www.nomadproject.io/docs/drivers/podman, see https://www.nomadproject.io/docs/nomad-vs-kubernetes

**Comment by vrothberg** [comment]
@ThorManager, Podman focuses on single-node environments only. It is not intended to run in an orchestrated fashion. It does however support running K8s yaml files:
* https://www.redhat.com/sysadmin/podman-play-kube-updates

**Comment by rhatdan** [comment]
Podman will never support Swarm functionality. If other tools are based on top of Podman to provide cross node functionality then that is fine, but we have no thoughts of adding this functionality to Podman.
